src/shared/components/kakaomap/searchCenterMap.tsx

```
@@ -0,0 +1,44 @@
+// src/components/SearchCenterMap.tsx
+import { Map, MapMarker } from 'react-kakao-maps-sdk';
```

Biome (2.1.2)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)